### PR TITLE
[strings] Add Azerbaijani translations

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

**Comment by pastk** [comment]
Could need to add `az` to a couple more files, please check
- #2247
- #2587 

namely `coding/string_utf8_multilang.cpp`, but maybe somewhere else

**Comment by pastk** [comment]
Somehow the `af` lang is missing in `indexer/categories_holder.hpp` (so we can't add `af` strings to categories.txt), could you please add it together with `az`? 
Ideally need to check if some other langs are missing too and re-sync.

**Comment by biodranik** [comment]
@pastk please don't add `af` yet into categories. Someone needs to translate most of the strings first. It's included in autogenerated translations, but not in categories.

**Review by pastk — APPROVED** [review]

File: coding/string_utf8_multilang.cpp [hunk 1]
```diff
@@ -40,7 +40,7 @@ std::array<StringUtf8Multilang::Lang, StringUtf8Multilang::kMaxSupportedLanguage
      {"uk", "Українська", {"Ukrainian-Latin/BGN"}},
      {"ca", "Català", {}},
      {"hu", "Magyar", {}},
-     {StringUtf8Multilang::kReservedLang /* hsb */, "", {}},
+     {"az", "Azərbaycanca", {"Azerbaijani-Latin/BGN"}},
```
[the remark this answers is not in the document]
Comment (biodranik) [reply coding/string_utf8_multilang.cpp:43]: 1. Please check ICU transliteration docs for the correct/supported ids.
2. Please check it on the device with your changes and make sure that it works and not crashes.

File: tools/python/translate.py [hunk 2]
```diff
@@ -114,6 +124,14 @@ def deepl_translate_one(text, source_language, target_language):
   json = response.json()
   return json['translations'][0]['text']
 
+def translate_one(text, source_language, target_language):
```
Comment (biodranik) [inline tools/python/translate.py:127]: These changes should be in a separate PR/commit.

File: tools/python/strings_utils.py [hunk 3]
```diff
@@ -484,6 +484,15 @@ def validate(self, langs=None):
         self.print_strings_with_wrong_placeholders(langs=args.langs)
         return not self.validation_errors
 
+    def translate(self, source_language, target_language):
```
Comment (biodranik) [inline tools/python/strings_utils.py:487]: misc script changes should be in a separate PR

**Review by biodranik — COMMENTED** [review]
Does it work on the device?
